pkgs/misc/leap-motion/default.nix

```diff
@@ -0,0 +1,90 @@
+{ stdenv, lib, fetchurl, dpkg, autoPatchelfHook
+, glib, dbus, libX11, libGL, alsaLib, zlib, xorg, libxslt  
+, libxml2, sqlite, libxcb, fontconfig, freetype, libGLU
+, gtk2-x11, atk, gtk2, gdk-pixbuf, pango, cairo }:
+
+stdenv.mkDerivation rec {
+  pname = "leap-motion";
+  version = "2.3.1";
+
+  # non-free binaries
+  src = fetchurl {
+    url = "https://warehouse.leapmotion.com/apps/4186/download";
+    sha256 = "1m3b029734yzhqbi78q36fi59xm9qrf05djshf059hq6ky9qahz0";
+  };
+
+  buildInputs = [
+    glib
+    dbus
+    libGL
+    libGLU
+    alsaLib
+    zlib
+    stdenv.cc.cc.lib
+    xorg.libXrender
+    xorg.libXext
+    xorg.libXi
+    xorg.libSM
+    xorg.libICE
+    libX11
+    libxslt
+    libxml2
+    sqlite
+    libxcb
+    fontconfig
+    gtk2-x11
+    atk
+    gtk2
+    gdk-pixbuf
+    pango
+    cairo
+  ]; 
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    dpkg
+  ];
+
+  unpackPhase = ''
+    tar xvf $src 
+    dpkg-deb -x Leap_Motion_Installer_Packages_release_public_linux/Leap-2.3.1+31549-x64.deb .
```

Comment:
This is going to break next time someone updates the src and version. Perhaps use a glob here?

Reply:
Fixed!